src/main/java/BattleshipModel.java

```diff
@@ -62,7 +63,7 @@ public void fire(Coordinate shot){
 
     }
 
-    public void AIFire(Board target)
+    public void aiFireHard(Board target)
```

Comment:
Thanks for renaming the function. This should clear up any confusion with the function declared above.